### ISSUE TITLE
Add a flatblock above "get started" on taxbrain input

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#176]
@feenberg pointed out that it is frustrating that the "get started" link takes the user to all of the introductory text on the TaxBrain input page. Could we have a new flatblock above the get started section, and I'll move that preamble up there? I'll leave the user tips in the existing flatblock under the get started section. @mistakevin, do you have bandwidth for this?

**Comment by talumbau** [comment]
I can take this so @mistakevin can focus on the dynamic landing page "construction zone" ideas plus the CSS issues on the new processing results

**Comment by MattHJensen** [comment]
@talumbau, I realize I made a blunder on this request. The intent is for the "get started" sidebar link to take the user to "Get Started" section of the page. I thought that is what was happening, so I requested a flat block above the "Get Started" section, but in fact the "get started" sidebar link takes the user to the top of the page. 

If the "get started" sidebar link were redirected to take the user to the "Get Started" section, then everything would work as desired now.
